pyglmnet/pyglmnet.py

```diff
@@ -138,15 +140,13 @@ class GLM(object):
 
     def __init__(self, distr='poisson', alpha=0.5,
                  Tau=None, group=None,
-                 reg_lambda=None,
+                 reg_lambda=np.logspace(np.log(0.5), np.log(0.01), 10,
+                                        base=np.exp(1)),
```

Comment:
... and here it is. This is `ndarray`, mutable. Actually turn this into a `tuple` of values (and any other offending ones) and you should be good to go